stdlib/time/src/main/ballerina/time/timelib.bal

```diff
@@ -32,115 +31,122 @@ public type Timezone record {
-    public extern function minute() returns (int);
-
-    # Returns the second representation of the given time.
-    #
-    # + return - The second-of-minute, from 0 to 59
-    public extern function second() returns (int);
-
-    # Returns the millisecond representation of the given time.
-    #
-    # + return - The milli-of-second, from 0 to 999
-    public extern function milliSecond() returns (int);
-
-    # Returns the date representation of the given time.
-    #
-    # + return - The year representation.
-    #            The month-of-year, from 1 (January) to 12 (December).
-    #            The day-of-month, from 1 to 31.
-    public extern function getDate() returns (int, int, int);
-
-    # Returns the time representation of the given time.
-    #
-    # + return - The hour-of-day, from 0 to 23.
-    #            The minute-of-hour to represent, from 0 to 59.
-    #            The second-of-minute, from 0 to 59.
-    #            The milli-of-second, from 0 to 999.
-    public extern function getTime() returns (int, int, int, int);
-
-    # Add specified durations to the given time value.
-    #
-    # + years - The year representation
-    # + months - The month-of-year to represent, from 1 (January) to 12 (December)
-    # + days - The day-of-month to represent, from 1 to 31
-    # + hours - The hour-of-day to represent, from 0 to 23
-    # + minutes - The minute-of-hour to represent, from 0 to 59
-    # + seconds - The second-of-minute to represent, from 0 to 59
-    # + milliSeconds - The milli-of-second to represent, from 0 to 999
-    # + return - Time object containing time and zone information after the addition
-
-    public extern function addDuration(int years, int months, int days, int hours, int minutes, int seconds,
-                                       int milliSeconds) returns (Time);
-
-    # Subtract specified durations from the given time value.
-    #
-    # + years - The year representation
-    # + months - The month-of-year to represent, from 1 (January) to 12 (December)
-    # + days - The day-of-month to represent, from 1 to 31
-    # + hours - The hour-of-day to represent, from 0 to 23
-    # + minutes - The minute-of-hour to represent, from 0 to 59
-    # + seconds - The second-of-minute to represent, from 0 to 59
-    # + milliSeconds - The milli-of-second to represent, from 0 to 999
-    # + return - Time object containing time and zone information after the subtraction
-    public extern function subtractDuration(int years, int months, int days, int hours, int minutes, int seconds,
-                                            int milliSeconds) returns (Time);
-
-    # Change the timezone of the given time.
-    #
-    # + zoneId - The new timezone id
-    # + return - Time object containing time and zone information after the conversion
-    public extern function toTimezone(string zoneId) returns (Time);
+public type Time record {
+    int time;
+    Timezone zone;
```

Comment:
Not related to a change introduced by this PR, but should we change "Timezone" to "Time**Z**one"?

Reply:
+1, thought of the same, but missed to do the change :)